ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/interfaces.j2

```diff
@@ -14,6 +15,8 @@
        or vxlan_interface is arista.avd.defined %}
 
 ## Interfaces
+{## System l1 #}
```

Comment:
IMO this should be under a different section. Something with hardware or system settings.

Reply:
Yeah, I debated with myself on this one... I put it in this section because it's about the "Configuration of L1 switch parameters". You can configure parameters for Modules, linecards, and interfaces. But open to suggestions!

Reply:
"System Settings" was my suggestion.